engine/jobmaster/dm/dm_jobmaster.go

```diff
@@ -321,14 +321,21 @@ func (jm *JobMaster) getInitStatus() ([]runtime.TaskStatus, []runtime.WorkerStat
 func (jm *JobMaster) preCheck(ctx context.Context, cfg *config.JobCfg) error {
 	jm.Logger().Info("start pre-checking job config")
 
+	// TODO: refactor this, e.g. move this check to checkpoint agent
```

Comment:
DM's op logic should also include this check. I have changed "close #xxx" to "ref #xxx" in PR description